src/index.js

```diff
@@ -58,6 +58,9 @@ import handleStartupEvent from './squirrel';
     mainWindow = new BrowserWindow(generateBrowserConfig());
     global.mainWindowID = WindowManager.add(mainWindow, 'main');
 
+    // Required for 'darwin'
+    mainWindow.setFullScreenable(true);
```

Comment:
Ah sorry, didn't have an up-to-date copy. Electron docs say by default it's false, but removing it seems to negate the need for this line. I was under the impression that setting it to true would open the app in fullscreen.